### PR TITLE
fix: docs: Modify generate-lotus-cli.py to ignoring aliases.

### PR DESCRIPTION
**Pull request by rjan90**
## Proposed Changes
Modify the generate-lotus-cli.py script to generate documentation for the primary sub-commands, ignoring any aliases. Previously the script would fail to generate CLI-documentation for commands with aliases, so we only would get an empty output: https://github.com/filecoin-project/lotus/blob/master/documentation/en/cli-lotus.md#lotus-state-sector-sector-info

Also add lotus-provider to the CLI-documentation generation script.

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] If the PR affects users (e.g., new feature, bug fix, system requirements change), update the CHANGELOG.md and add details to the UNRELEASED section.
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: documentation/en/cli-lotus-provider.md [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+OPTIONS:
+   --deadline value                   deadline to compute WindowPoSt for  (default: 0)
+   --layers value [ --layers value ]  list of layers to be interpreted (atop defaults). Default: base (default: "base")
+   --help, -h                         show help
+```
+
+## lotus-provider version
+```
+NAME:
+   lotus-provider version - Print version
```
Comment (snadrus) [inline documentation/en/cli-lotus-provider.md:253]: This does not exist.

File: documentation/en/cli-lotus-provider.md [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+USAGE:
+   lotus-provider version [command options] [arguments...]
+
+OPTIONS:
+   --help, -h  show help
+```
+
+## lotus-provider auth
+```
+NAME:
```
Comment (snadrus) [inline documentation/en/cli-lotus-provider.md:264]: auth does not exist.

File: documentation/en/cli-lotus-provider.md [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+   lotus-provider auth api-info [command options] [arguments...]
+
+OPTIONS:
+   --perm value  permission to assign to the token, one of: read, write, sign, admin
+   --help, -h    show help
+```
+
+## lotus-provider log
+```
+NAME:
```
Comment (snadrus) [inline documentation/en/cli-lotus-provider.md:307]: log does not exist

File: documentation/en/cli-lotus-provider.md [hunk 4]
```diff
@@ -0,0 +1,410 @@
...
+   lotus-provider log alerts [command options] [arguments...]
+
+OPTIONS:
+   --all       get all (active and inactive) alerts (default: false)
+   --help, -h  show help
+```
+
+## lotus-provider wait-api
+```
+NAME:
```
Comment (snadrus) [inline documentation/en/cli-lotus-provider.md:383]: wait does not exist

**Review by snadrus — CHANGES_REQUESTED** [review]

File: documentation/en/cli-lotus-provider.md (hunk 2 above)
Comment (rjan90) [reply documentation/en/cli-lotus-provider.md:264]: It does seem like we inherit `auth` from somewhere at least:

Screenshot 2023-12-18 at 17 48 52

**Review by snadrus — APPROVED** [review]

**Pull request merged**